internal/globalMerge.ecl

```diff
@@ -155,13 +155,13 @@ EXPORT globalMerge := MODULE
               id = id - lastgroupend;
               pid = pid - lastgroupend;
               *(uint32_t *)pos = wi;
-              pos += sizeof(uint32_t);
+              pos =(uint8_t *)pos + sizeof(uint8_t);
```

Comment:
No need this, keep the original code 'pos += sizeof(uint32_t);' here and all further positions.

Reply:
This makes sense. I made the changes accordingly. The code complied and completed without issues. Please review. Thank you. @ghalliday